tests/mir-opt/copy-prop/custom_move_arg.rs

```diff
@@ -0,0 +1,32 @@
+// unit-test: CopyProp
+
+#![feature(custom_mir, core_intrinsics)]
+#![allow(unused_assignments)]
+extern crate core;
+use core::intrinsics::mir::*;
+
+struct NotCopy(bool);
+
+// EMIT_MIR custom_move_arg.f.CopyProp.diff
+#[custom_mir(dialect = "analysis", phase = "post-cleanup")]
+fn f(_1: NotCopy) {
+    mir!({
+        let _2 = Move(_1);
```

Comment:
```suggestion
        let _2 = _1;
```
Otherwise, we'd already have a use-after-move in the input MIR.